docs/source/blog/displacement-vectors.md

```diff
@@ -0,0 +1,47 @@
+---
+blogpost: true
+date: October 27, 2025
+author: Carlo Castoldi
+location: Milan, Italy
+category: update
+language: English
+---
+
+# Improving displacement vectors
+
+Computing locomotion features is a core utility since early versions of `movement`, when they where first introduced by [@lochhh](https://github.com/lochhh) in [#106](https://github.com/neuroinformatics-unit/movement/pull/106) for [v0.0.13](https://github.com/neuroinformatics-unit/movement/releases/tag/v0.0.13)—we are now at [v0.10.0](https://github.com/neuroinformatics-unit/movement/releases/tag/v0.10.0).
+For a library focused on tracking analysis and motion quantification, it's essential that key information, such as the total distance traveled or the tracking point's location in adjacent frames (next or previous), can be easily calculated. That's why we introduced {func}`compute_displacement <movement.kinematics.compute_displacement>` so early on: it's a function that some of you may have used to have a grasp of your tracking data, and it is also featured in our {ref}`compute and visualise kinematics <sphx_glr_examples_compute_kinematics.py>` example<!--#compute-displacement-vectors heading-->.
+
+Its implementation, however, produces results of complex interpretability. It defines, for a given individual and keypoint at timestep `t`, a vector that points to the opposite direction of its previous position at time `t-1`.
+
+For this reason, during the Hackday for [OSW 2025](https://neuroinformatics.dev/open-software-summer-school/2025/index.html) and as my first contribution to `movement`, I [offered myself](https://github.com/neuroinformatics-unit/osw25-hackday/issues/16) to work on a more intuitive interface for displacement vectors, supervisioned by [@sfmig](https://github.com/sfmig).
+In an inspiring collaborative effort to simplify the user experience, we are pleased to present the key changes introduced in [#657](https://github.com/neuroinformatics-unit/movement/pull/657). This update provides an elegant, well-tested, and thoroughly documented solution that reduces the mental load on users.
+
+__API changes__
```

Comment:
Do you think this section would benefit from adding an image like the one below? If you think it helps I'm happy to push it to the PR, after you're done implementing my other suggestions.

![](image.png)

Reply:
Hey @carlocastoldi, do you think you'll be able to make the final pass on this within the week?
I think we're very close to being done here. Let me know what you think of my modified figure and the other couple of unresolved comments.